Utilities/gitversion.h

```diff
@@ -0,0 +1 @@
+const char *gitversion = "864d7e9bc2d69154f4f773f4d37cb1e6daa1c9f1";
```

Comment:
We should never push modules.h or gitversion.h

Reply:
Ok, can we have a .gitignore in the repo?